.github/workflows/sync-package-periodically.yml

```diff
@@ -35,12 +44,31 @@ jobs:
       - name: Rebuild and validate website
         run: scripts/rebuild.sh
 
-      - name: Commit Results
+      - name: Commit and Push changes
         run: |
             git config --local user.email "actions@github.com"
             git config --local user.name "GitHub Actions"
-            git add scripts/commit.txt
-            git commit -m 'Data Updated' ||  echo 'No changes to commit'
-            git fetch origin main
-            git push origin HEAD:main || echo "Error occurred while push to origin"
-            
+            git add .
+            git commit -m 'Update package metadata and website content' || echo 'No changes to commit'
+            git push -f origin package_sync || echo "Error occurred while push to origin"
```

Comment:
These two statements will eat the failures and continue on to opening the pull request.

I think it would be better to drop the `|| echo` statements and simply let git speak for itself.